src/Adapter/Admin/NotificationsConfiguration.php

```diff
@@ -48,9 +48,9 @@ public function __construct(Configuration $configuration)
     public function getConfiguration()
     {
         return array(
-            'show_notifs_new_orders' => $this->configuration->get('PS_SHOW_NEW_ORDERS'),
-            'show_notifs_new_customers' => $this->configuration->get('PS_SHOW_NEW_CUSTOMERS'),
-            'show_notifs_new_messages' => $this->configuration->get('PS_SHOW_NEW_MESSAGES'),
+            'show_notifs_new_orders' => (bool)$this->configuration->get('PS_SHOW_NEW_ORDERS'),
```

Comment:
we shouldn't have to cast values, if its not boolean values lets change them in database.

Reply:
I had to, as the Configuration table allows every primitive types to be stored.